service/src/main/resources/schemas/test.schema.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This shouldn't be in the `src/main/resources` directory, it should be in the `src/test/resources` directory, since it's a resource only used by tests and is not part of the deployed service.